app/controllers/movies_controller.rb

```diff
@@ -1,10 +1,13 @@
 class MoviesController < ApplicationController
 
   before_filter :get_genres
+  before_action :authenticate_user!
 
 
   def index
+    #@movies = Movie.includes(:user_movies).all
```

Comment:
Remove comment. In general we dont leave comments of code in repo on Passare. Only in rare cases if it is commented temporary and it will be returned in future.